pack-contracts/pack-contract-grpc/src/main/proto/GrpcTxEvent.proto

```diff
@@ -42,9 +42,13 @@ message GrpcTxEvent {
   bytes payloads = 7;
   string serviceName = 8;
   string instanceId = 9;
-  int32 forwardTimeout = 10;
-  int32 forwardRetries = 11;
-  string retryMethod = 12;
+  int32 timeout = 10;
```

Comment:
We need to mention the change of GrpcTxEvent.proto in the release note,  to notify other language client for updating.